OpenSim/Simulation/Test/testRealizeDynamics.cpp

```diff
@@ -0,0 +1,101 @@
+/* -------------------------------------------------------------------------- *
+ *                    OpenSim:  testRealizeDynamics.cpp                       *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
+ * Author(s): Thomas Lau                                                      *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+#include <stdint.h>
+#include <OpenSim/Simulation/Manager/Manager.h>
+#include <OpenSim/Simulation/Control/ControlSetController.h>
+#include <OpenSim/Simulation/Model/Model.h>
+#include <OpenSim/Common/LoadOpenSimLibrary.h>
+#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
+
+using namespace OpenSim;
+using namespace std;
+
+//==============================================================================
+// testRealizeDynamics tests that the results produced in every stage are 
+// repeatable - this test is designed primarily to make sure that there are no 
+// memory races during realizeDynamics that can cause time-dependent results 
+// with regards to thread timing.
+//==============================================================================
+void testRealizeDynamics(const string& modelFile);
+
+static const int MAX_N_TRIES = 100;
+
+int main()
+{
+    try {
+        LoadOpenSimLibrary("osimActuators");
```

Comment:
Can you try removing this? I suspect you copied this from another test case. I think it is in general not necessary. If it is necessary, it would only be necessary on Windows, and then appveyor would fail. Oh I guess we'd have to wait until we have a first functional build on appveyor to test this...